libs/shared/ui/src/lib/components/navigation-left/navigation-left-sub-link/navigation-left-sub-link.spec.tsx

```diff
@@ -10,90 +10,46 @@ jest.mock('react-router-dom', () => ({
 
 describe('NavigationLeftSubLink', () => {
   const props: NavigationLeftSubLinkProps = {
-    linkContent: jest.fn(),
     link: {
       title: 'my-title',
-      icon: 'icon-solid-angle-down',
+      icon: 'icon-solid-play',
       url: '/general',
-      onClick: jest.fn(),
       subLinks: [
         {
           title: 'title',
           url: '/general-second',
-          onClick: jest.fn(),
         },
       ],
     },
-    linkClassName: jest.fn(),
   }
 
   it('should render successfully', () => {
-    const { baseElement } = render(<NavigationLeftSubLink {...props} />)
+    const { baseElement } = renderWithProviders(<NavigationLeftSubLink {...props} />)
     expect(baseElement).toBeTruthy()
   })
 
-  it('should open sub links', () => {
-    render(<NavigationLeftSubLink {...props} />)
+  it('should open sub links', async () => {
+    const { userEvent } = renderWithProviders(<NavigationLeftSubLink {...props} />)
 
     const trigger = screen.getByTestId('link')
 
-    fireEvent.click(trigger)
+    await userEvent.click(trigger)
 
     const subLinks = screen.getByTestId('sub-links')
 
     expect(subLinks).toBeInTheDocument()
   })
 
-  it('should have rotate the arrow', () => {
-    render(<NavigationLeftSubLink {...props} />)
+  it('should have rotate the arrow', async () => {
+    const { userEvent } = renderWithProviders(<NavigationLeftSubLink {...props} />)
 
     const trigger = screen.getByTestId('link')
 
-    fireEvent.click(trigger)
-
+    console.log(trigger.querySelector('.icon-solid-angle-down')?.classList.toString())
```

Comment:
need to be remove

Reply:
Fixed https://github.com/Qovery/console/pull/1660/commits/6363b67a78181dfc46f2ffe6984a8d76594a0863